### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by SAZZAD1Q2**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+body {
+  /* background-color: #f5f6f8; */
+  background-image: linear-gradient(#538252, #64c674);
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+  color: #888;
+}
+
+a {
+  color: #000;
```
Comment (mikethreels) [inline app/assets/stylesheets/application.css]: - [ ] please make sure to use the color scheme provided in the template:
image

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi @SAZZAD1Q2, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job creating the project

## Required Changes ♻️
- [ ] One of the key requirements is that your project has a note in the PR which describes the changes made in this milestone.
- [ ] According to the requirements there should be a live demo deployed and there should also be a short demonstration video.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -61,231 +24,141 @@ For more information on the importance of a professional README for your reposit
...
-```
---->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Deployment
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
 
-You can deploy this project using:
+- [Live Demo Link](https://mobile-app-zn.onrender.com)
```
Comment (VICTOR-CHUKWU) [inline README.md:106]: - [ ] I really don't know what is going on but the app you have live is different from what I see  locally, maybe you deployed the wrong repo, kindly check 
your local app 
Screenshot from 2023-10-20 16-54-28
you live app
Screenshot from 2023-10-20 16-55-46

File: README.md [hunk 3]
```diff
@@ -61,231 +24,141 @@ For more information on the importance of a professional README for your reposit
...
-Example:
+<p align="right">(<a href="#readme-top">back to top</a>)</p
 
-```sh
+<!-- Presentation Video -->
+## 🎥 Presentation Video <a name="presentation-video"></a>
 
-```
- -->
+- [Presentation Video Link](https://www.loom.com/share/590eccee11f94d65b2fb7aaddeaedc70?sid=1d276fce-a84b-4b5e-9db2-02254369e9f5)
```
Comment (VICTOR-CHUKWU) [inline README.md:113]: - [ ] In your video presentation you did not show us a part of your code you are proud of, kindly include that

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <% end %>
+<% end %>
+
+</div>
+  <div class="position-fixed bottom-0 w-100">
+    <div class="d-flex justify-content-center align-items-center bg-white p-2">
+
+  <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary w-full w-50' %>
+</div>
+</div>
```
Comment (VICTOR-CHUKWU) [inline app/views/home/index.html.erb:56]: - [ ] kindly try to match your page to the given wireframe
your app 
Screenshot from 2023-10-20 16-55-40
given wireframe
Screenshot from 2023-05-06 13-54-49

File: app/views/entities/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+  
+    </div>
+ 
+<section class="auth m-3 shadow-sm">
+
+<% if @entity.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(@entity.errors.count, "error") %> prohibited this model from being saved:</h2>
+    <ul>
```
Comment (VICTOR-CHUKWU) [inline app/views/entities/new.html.erb:29]: - [ ] when i try to add  a transaction I get an error kindly fix
Screenshot from 2023-10-20 17-05-24

File: README.md [hunk 6]
```diff
@@ -61,231 +24,141 @@ For more information on the importance of a professional README for your reposit
...
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+I would like to thank Microverse community.
```
Comment (VICTOR-CHUKWU) [inline README.md:157]: - [ ] The README  should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @SAZZAD1Q2 , 
## STATUS: CHANGES REQUIRED :recycle: 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (SAZZAD1Q2) [reply README.md:106]: Thank you for your review. You are right. I made a fatal mistake when I made my pr for review. I will resolve it and submit again for review. 🥇

**Pull request merged**